src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -410,11 +483,67 @@ std::string GcsResourceManager::ToString() const {
   std::string indent_1(indent + 1 * 2, ' ');
   ostr << "{\n";
   for (const auto &entry : cluster_scheduling_resources_) {
-    ostr << indent_1 << entry.first << " : " << entry.second->DebugString() << ",\n";
+    ostr << indent_1 << entry.first << " : " << entry.second->GetLocalView().DebugString()
+         << ",\n";
   }
   ostr << indent_0 << "}\n";
   return ostr.str();
 }
 
+void GcsResourceManager::DeleteResources(NodeResources *node_resources,
```

Comment:
Move this function back to the original position? This will make review easier and help keep a clean git history.

Reply:
Okey, I will do it.